src/combine.cpp

```diff
@@ -500,11 +500,11 @@ static chunk_t *get_d_template_types(ChunkStack &cs, chunk_t *open_paren);
 static bool chunkstack_match(ChunkStack &cs, chunk_t *pc);
 
 
-enum PLBfound
+enum class plb_found_e : uint8_t
```

Comment:
It keeps the state of the sequence of found angle brackets inside an objective-c class deceleration/definition  (not sure if both or just one of them). I would call it oc-class_angle_state.
`PLBfound` is defined in _combine.cpp_ but only used for `handle_oc_class()`. I would __move__ it into that function and name it `angle_state_e`.